app/models/user.rb

```diff
@@ -18,4 +18,11 @@ class User < ActiveRecord::Base
   def cards_for_review
     current_deck ? current_deck.cards.for_review : cards.for_review
   end
+
+  def self.notify_pending_cards
+    joins(:cards).where("cards.review_date <= ?", Time.current).group("users.id")
+                 .select("users.*, count(cards.id) as count_card").each do |user|
```

Comment:
Line is too long. [81/80]<br>Place the . on the previous line, together with the method call receiver.<br>Use 2 (not 13) spaces for indenting an expression spanning multiple lines.